specification/cdn/resource-manager/Microsoft.Cdn/2016-10-02/cdn.json

```diff
@@ -343,6 +343,47 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cdn/profiles/{profileName}/getSupportedOptimizationTypes": {
```

Comment:
Each profile has a SKU associated with it like Standard_Akamai, Premium_Verizon etc and we need to update the supported optimization types for a SKU from time to time based on the features provided by the providers. Therefore, if we have it as a property of a profile, we'll need to update all the profiles in the datastore every time we update the supported optimization types for a SKU. So, to keep it dynamic we have exposed the method 'GetSupportedOptimizationTypes' for profiles.